src/_cffi_src/openssl/crypto.py

```diff
@@ -57,6 +58,16 @@
 
 /* This was removed in 1.1.0 */
 void CRYPTO_lock(int, int, const char *, int);
+
+/* Signature changed significantly in 1.1.0, only expose there for sanity */
```

Comment:
This comment is confusing to me -- I think you mean we only expose it on 1.1.0+ and conditionally remove it on anything older?